packages/language-server/src/languageServerPlugin.ts

```diff
@@ -75,8 +75,7 @@ export const plugin: LanguageServerPlugin = (
 					languages: ['astro'],
 					ignoreIdeOptions: true,
 					resolveConfigOptions: {
-						// Prettier's cache is a bit cumbersome, because you need to reload the config yourself on change
-						// TODO: Upstream a fix for this
+						// This seems to be broken since Prettier 3, and it'll always use its cumbersome cache. Hopefully it works one day.
```

Comment:
Prettier 3 seems to have a weird caching issue. I tried everything: Setting `useCache` to `false`, calling `prettier.clearConfigCache()` manually etc and nothing worked, it always uses a cached config. This is only annoying if you're updating your Prettier config often, which hopefully doesn't happen too much.